### ISSUE TITLE
Only display errors if field has been modified

### ISSUE THREAD

**Issue by depiction** [Deadly0/final-form-material-ui#29]
If a field is set to auto focus and a user moves away from the field without making any changes, validation runs and errors display. I think that errors shouldn't be displayed until a field has been modified. Final Form provides the modified meta value, therefore this should be an easy change.

**Comment by lookfirst** [comment]
@depiction This project does not seem to be developed any further. If you'd like to try [MUI-RFF which is maintained](https://github.com/lookfirst/mui-rff), I invite you. I'm not 100% sure the logic is what you want, so please try it out and let me know over there.

**Comment by depiction** [comment]
Yeah, I assumed that this project was dead, despite the fact that React Final Form still uses this in their Material UI demo form. Thanks for suggesting MUI-RFF. I didn't see it come up in any of my searches. However, after looking at the code for it, it has this same problem. I ended up copying the components from this project into custom components then making the necessary changes.

**Comment by lookfirst** [comment]
I updated the RFF site (and MUI) to link to MUI-RFF in the docs, so it has both options for discovery. Yes, SEO has been hard on this one for some reason.

If you'd like to discuss over on MUI-RFF, I am open to adding these sorts of features so you don't have to maintain your own fork.
